snapcraft/plugins/v1/catkin.py

```diff
@@ -837,25 +848,28 @@ def snap_fileset(self):
         return fileset
 
 
-def _find_system_dependencies(catkin_packages, rosdep, catkin):
+def _find_system_dependencies(self, rosdep, catkin):
```

Comment:
This indent is wrong (to move this method into the class).

Reply:
Fixed.

Reply:
(we wouldn't want to do that anyway, fix looks good)